sdcm/cluster.py

```diff
@@ -2367,7 +2367,7 @@ def run_stress_thread(self, stress_cmd, timeout, output_dir, stress_num=1, keysp
 
         if keyspace_name:
             stress_cmd = stress_cmd.replace(" -schema ", " -schema keyspace={} ".format(keyspace_name))
-        elif 'keyspace=' not in stress_cmd: # if keyspace is defined in the command respect that
+        else:
```

Comment:
We won't assign keyspace in original stress cmdline, your change is fine.

Can we safely add an assert in else block?

```
        if keyspace_name:
            stress_cmd = stress_cmd.replace(" -schema ", " -schema keyspace={} ".format(keyspace_name))
        else:
            assert('keyspace=' not in stress_cmd)
            .....
```

Reply:
This change was intentionally done by me, to enable putting the keyspace in the stress commands.
I believe it was removed by mistake by alex.

Reply:
Not sure was done by me. may some how leaked from pr 814. Restore